torch/autograd/profiler_util.py

```diff
@@ -252,7 +262,37 @@ def export_chrome_trace(self, path):
                         f'"tid": {evt.thread}, '
                         '"pid": "CPU functions", '
                         f'"id": {next_id}, '
-                        f'"cat": "cpu_to_{device_name}", '
+                        '"cat": "cpu_to_cuda", '
```

Comment:
There is an advantage in using only `cpu_to_cuda` for all flows including xpu and privateuse1. This would prevent keeping up-to-date the names in downstream analyzers such as HTA (https://github.com/facebookresearch/HolisticTraceAnalysis) cc @briancoutinho , @anupambhatnagar